core/src/test/java/io/grpc/internal/ManagedChannelServiceConfigTest.java

```diff
@@ -163,7 +163,7 @@ public void managedChannelServiceConfig_parseMethodConfig() {
             .put("backoffMultiplier", 1.5D)
             .put("perAttemptRecvTimeout", "2.5s")
             .put("retryableStatusCodes", ImmutableList.of("UNAVAILABLE"))
-            .build());
+            .buildOrThrow());
```

Comment:
There are 4 more occurrences in this file: lines 228, 245, 262 and 288. Why were they not changed? Unless there is a reason to only change some and not all.